test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -23,12 +23,11 @@ describe(':terminal', function()
     ]])
     -- Invoke a command that emits frequent terminal activity.
     execute([[terminal while true; do echo X; done]])
-    helpers.feed([[<C-\><C-N>]])
     screen:expect([[
+      ^X                                                 |
       X                                                 |
       X                                                 |
-      ^X                                                 |
-                                                        |
+      :terminal while true; do echo X; done             |
```

Comment:
If CI passes with your changes here, let's leave this screen test even though I removed it on master.